wandb/util.py

```diff
@@ -70,6 +70,27 @@
 else:
     SENTRY_ENV = "production"
 
+
+PLATFORM_WINDOWS = "windows"
+PLATFORM_LINUX = "linux"
+PLATFORM_BSD = "bsd"
+PLATFORM_DARWIN = "darwin"
+PLATFORM_UNKNOWN = "unknown"
```

Comment:
nit: feel free to ignore as it was done already in this file but I kinda think these module globals are an anti-pattern.

It is probably in the noise with how much other stuff is doing this in the library, but it means that these statements will be executed on import.   So imagine you had 1000 possible platforms, you are now initializing 1000 of these at import time.... only to later possibly use one of these.